atlp-rwanda/Avengers-Phantom-BE
# ft(Testing bus):Bus CRUD Operations  tests

### What does this PR do?

Testing Bus CRUD operations

### Description of Task to be completed?

 - [x] Test Post BUS API
 - [x] Test Get BUS API
 - [x] Test Get all BUSES API
 - [x] Test Update BUS API
 - [x] Test Delete BUS API
 - [x] Deployed Backend app to Heroku
 
### How should this be manually tested?
step1: clone git repo if not yet done
step2: cd Avengers-Phantom-BE
step3: npm i or npm install to install dependencies
step4: npm run start:dev to start the server
step5:npm run test to check all passing tests

### TESTED APIs
POST /api/v1/buses
GET /api/v1/buses
GET /api/v1/buses/:uuid
PATCH /api/v1/buses/:uuid
DELETE /api/v1/buses/:uuid
